shared/src/commonMain/sqldelight/com.jetbrains.spacetutorial.cache/AppDatabase.sq

```diff
@@ -0,0 +1,23 @@
+import kotlin.Boolean;
+
+CREATE TABLE Launch (
+    flightNumber INTEGER NOT NULL,
+    missionName TEXT NOT NULL,
+    details TEXT,
+    launchSuccess INTEGER AS Boolean DEFAULT NULL,
```

Comment:
Couldn't this one be not null as well? `Boolean DEFAULT NULL` seems error-prone.

(And then the data class could have a `Boolean` instead of a `Boolean?`)